packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/check-on-startup/react-native/checkfortoken.md

```diff
@@ -0,0 +1,10 @@
+Call the `isAuthenticated` function if you want to check whether the user is signed-in:
```

Comment:
no need to hyphenate signed-in

Call the `isAuthenticated` function if you want to check whether the user is signed in: